.editorconfig

```diff
@@ -36,11 +36,6 @@ indent_size = 2
 [*.xlf]
 indent_style = tab
 
-# SQL-Files
-[*.sql]
```

Comment:
What's the reason to remove this ?

This is the same setting which core has for sql files. Pull-request says to adjust to the current style, but settings do not match core style settings. Don't get this at all.

Or did I missed some style definitions ?

See: https://github.com/TYPO3/typo3/blob/813917860a7bbcecb27356bcbd0a631307ae8ec4/.editorconfig#L50

From my point-of-view I would say this should be reverted and the settings kept.

Reply:
styleguide currently indents 4 spaces on ext_tables.sql. Let's keep this removal since it now reflects the gives situation - and it's one special case less to take care of.